src/docdb/commands.ts

```diff
@@ -118,8 +119,18 @@ export class DocDBCommands {
         return undefined;
     }
 
+    private static validateDocumentID(name: string): string | undefined | null {
+        if (/^[-a-z0-9]+$/ig.test(name)) {
+            return "ID should contain only the following : digits, alphabets and -";
+        }
+        if (name.length > 40) {
```

Comment:
You are not checking lower boundary, does it mean empty strings are allowed as document IDs?

Reply:
Yes. It auto-generates an ID.